aldryn_forms/admin.py

```diff
@@ -98,14 +98,25 @@ def _clean_data(obj):
                 headers[ugettext('Language')] = 'language'
                 headers[ugettext('Submitted on')] = 'sent_at'
 
-                response = export(
-                    request,
-                    queryset=entries,
-                    model=entries.model,
-                    headers=headers,
-                    format='xls', # we can make this a field in the form ;)
-                    filename=filename
-                )
+                try:
```

Comment:
@yakky I would rewrite this as:

```python
from functools import partial

do_export = partial(
    export,
    request=request,
    queryset=entries,
    model=entries.model,
    headers=headers,
    filename=filename
)

try:
    # Since django-tablib 3.1 the parameter is called file_type
    response = do_export(file_type='xls')
except TypeError:
    response = do_export(format='xls')

````